cloud/amazon/sns_topic.py

```diff
@@ -228,9 +252,14 @@ def main():
         if not next_token:
             break
 
-    desired_subscriptions = [(sub['protocol'],
```

Comment:
Yes. Yes it would. This is why i shouldn't write my PRs while also watching Game of Thrones.